README.md

```diff
@@ -129,6 +129,7 @@ Use the following environment variables to configure the application:
 | <tt>OWNER_CHECKER_REPOSITORY</tt>  <b>*</b>    | | The owner and repository name separated by slash. For example, gh-codeowners/codeowners-samples. Used to check if GitHub owner is in the given organization.                                                                                                                                                                                                                                                                                                    |
 | <tt>OWNER_CHECKER_IGNORED_OWNERS</tt>          | `@ghost`| The comma-separated list of owners that should not be validated. Example: `"@owner1,@owner2,@org/team1,example@email.com"`.                                                                                                                                                                                                                                                                                                                                     |
 | <tt>OWNER_CHECKER_ALLOW_UNOWNED_PATTERNS</tt> | `true` | 	Specifies whether CODEOWNERS may have unowned files. For example: <br> <br>  `/infra/oncall-rotator/                    @sre-team` <br>  `/infra/oncall-rotator/oncall-config.yml` <br> <br>  The `/infra/oncall-rotator/oncall-config.yml` file is not owned by anyone.                                                                                                                                                                                           |
+| <tt>OWNER_CHEKER_OWNERS_MUST_BE_TEAMS</tt>     | `false` | Specifies whether only teams are allowed as owners of files |
```

Comment:
**nit:**
```suggestion
| <tt>OWNER_CHEKER_OWNERS_MUST_BE_TEAMS</tt>     | `false` | Specifies whether only teams are allowed as owners of files. |
```

Additionally, please add:
```yaml
  owner_cheker_owners_must_be_teams:
    description:  "Specifies whether only teams are allowed as owners of files."
    default: "false"
    required: false
```

to [action.yml](https://github.com/mszostok/codeowners-validator/blob/main/action.yml)

Reply:
sadly, this is not documented in the readme https://github.com/mszostok/codeowners-validator#checks=

Reply:
Thanks for letting me know, I will add it there too. For now, it was documented in configuration and in the release notes.

Reply:
no, thank you @mszostok :)